guestbook/README.md

```diff
@@ -55,6 +56,31 @@ If you see a url response, you are ready to go. If not, read the [Getting Starte
 
 All the files referenced in this example can be downloaded [from GitHub](https://git.k8s.io/examples/guestbook).
 
+### Building The Docker Images
```

Comment:
Looks like there is repetition with this heading and in the sentence below.

You can just change the capitalization to 

     ### Building the Docker images

and remove the following sentence: `Building docker images:`.